packages/launcher/lib/windows/index.ts

```diff
@@ -32,29 +32,20 @@ function formChromeCanaryAppPath () {
     'chrome.exe'
   )
 
-  return normalize(exe)
+  return [normalize(exe)]
 }
 
-function formFirefoxAppPath () {
-  const exe = 'C:/Program Files (x86)/Mozilla Firefox/firefox.exe'
-
-  return normalize(exe)
-}
-
-function formFirefoxDeveloperEditionAppPath () {
-  const exe = 'C:/Program Files (x86)/Firefox Developer Edition/firefox.exe'
-
-  return normalize(exe)
-}
-
-function formFirefoxNightlyAppPath () {
-  const exe = 'C:/Program Files (x86)/Firefox Nightly/firefox.exe'
-
-  return normalize(exe)
+function getFirefoxPaths (editionFolder) {
+  return () => {
+    return (['Program Files', 'Program Files (x86)'])
```

Comment:
will it be possible to install 32 and 64 bit Firefox - and then both will be detected and have same name and version and everything?

Reply:
yes, and no, 64-bit firefox will clobber 32-bit firefox (if both are installed)

this would be an exceptional case, though - in my tests, installing 32-bit will uninstall the 64-bit version, and vice versa. so they'd really have to be doin somethin crazy to run into this.